data/language_manager.bgt

```diff
@@ -0,0 +1,133 @@
+4) Cada mensaje dentro del archivo de idioma debe comenzar con el n�mero y el signo igual (=) para identificar el mensaje. El n�mero del primer mensaje debe ser el 1. Por ejemplo, 1=�Hola mundo!
+5) Cuando finalices el mensaje, debes presionar Enter y repetir el paso anterior hasta terminar con todos los mensajes en tu archivo.
+6) Para usar la librer�a en tu juego, incluye la librer�a como cualquier librer�a (#include) y en el void main() inicializa la funci�n escribiendo esto
+	language_manager("nombre_carpeta_idioma");
+El "nombre_carpeta_idioma" es la ruta de la carpeta que contiene los archivos de idioma.
+7) Si tienes dos o m�s archivos de idioma, se activa un men� para seleccionar un archivo de idioma. Y si tienes solo un archivo de idioma, se selecciona autom�ticamente sin activar el men�.
+8) Lo m�s importante: Para mostrar los mensajes en tu juego, llama al array voice[posicion], donde "posicion" es el n�mero del mensaje. Por ejemplo, voice[1];
+*/
+
+
+
+
+// Global variables. Variables Globales
+string[] voice; // The array that contents the messages. El array que contiene los mensajes
+sound menuclick, menuenter;
+
+// Language Manager Function. Funcionamiento del Gestor de idioma
+void language_manager(string folder_name)
+  {
+  int voice_size=0; // Total messages into language file (this is calculated automatically later). Total de mensajes en el archivo de idioma (se calcula autom�ticamente despu�s)
+  string filename; // The current language file used. Archivo de idioma usado actualemnte
+  file language; // File object. Objeto file
+  string[] language_files; // List of the language files. Lista de los archivos de idioma
+
+  language_files=find_files(folder_name+"\\*.*"); // Searching language files. Buscamos los arhivos de idioma
+  if(language_files.length()==0) // Files not found. No se encontraron los archivos
+    {
+    alert("Error", "Language files not found in directory "+folder_name+". Los archivos de idioma no se encontraron en el directorio "+folder_name);
+    exit();
+    }
+  if(language_files.length()==1) filename=folder_name+"\\"+language_files[0]; // Only 1 language file. Solo 1 archivo de idioma
+  if(language_files.length()>1) // Two or more language files. If it is, a menu is opened. Dos o m�s archivos de idioma. Si es as�, se abre un men�
+    {
+show_game_window("Select Language");
+menuclick.load("sounds/menuclick.wav");
+menuenter.load("sounds/menuenter.wav");
+    string[] menu_items(language_files.length()); // This array contains the name of language files. Este array contiene los nombres de los archivos de idioma
+    for(int i=0; i<language_files.length(); i++) // Loop to adding items into menu. Bucle para a�adir objetos al men�
+      {
+      menu_items[i]=language_files[i]; // Adding new item. A�adimos un nuevo objeto
+      wait(5); // Like to don't overload the CPUE. Bueno para no sobrecargar la CPU
+      }
+//tts_voice speech; // Voice object. Objeto de voz
```

Comment:
Is that no longer necessary?

Reply:
Is the coughing sound in the learn game sounds menu no longer necessary? I'm not sure; I kept a lot of the sounds there so that people wouldn't have to wonder what the sound meant. If you meant the above comment about the tts voice object, I just left the comments in place. I felt it was unnecessary, but I didn't think it would hurt anything to leave it.